webpack/assets/javascripts/react_app/components/Layout/components/InstanceTitleViewer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import NavItem from './NavItem';
+import { translate as __ } from '../../../common/I18n';
+
+const InstanceTitleViewer = ({
+  title,
+}) => {
+  if (title == undefined || title == "") {
```

Comment:
I think you can have 
```js
if (!title)
```
as they are both falsy values.

Reply:
Fixed. Thank you!